### PR TITLE
Hb 1

### PR DESCRIPTION
**Pull request by RuslanIbIsh**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -7,6 +7,86 @@
...
+                        <source>${jdk.version}</source>
+                        <target>${jdk.version}</target>
+                        <encoding>${project.build.sourceEncoding}</encoding>
+                    </configuration>
+                </plugin>
+            </plugins>
+        </pluginManagement>
+    </build>
 
 </project>
```
Comment (Ilya-brinchuk) [inline pom.xml:92]: Should be empty line

**Review by Ilya-brinchuk — APPROVED** [review]

File: pom.xml (hunk 1 above)
Comment (RuslanIbIsh) [reply pom.xml]: done

File: src/main/java/com/iri/movietickets/dao/impl/MovieDaoImpl.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+            session.save(movie);
+            transaction.commit();
+            return movie;
+        } catch (Exception e) {
+            if (transaction != null) {
+                transaction.rollback();
+            }
+            throw new DataProcessingException("Could not insert entity" + movie, e);
+        } finally {
+            session.close();
```
Comment (Sofasmile) [inline src/main/java/com/iri/movietickets/dao/impl/MovieDaoImpl.java:31]: let's add check for `null` value

**Review by Sofasmile — CHANGES_REQUESTED** [review]

File: src/main/java/com/iri/movietickets/dao/impl/MovieDaoImpl.java (hunk 2 above)
Comment (RuslanIbIsh) [reply src/main/java/com/iri/movietickets/dao/impl/MovieDaoImpl.java]: done

**Review by RomanDubovskyi — APPROVED** [review]

**Pull request merged**